### ISSUE TITLE
Shared notes editor breaks when entering too long texts

### ISSUE THREAD

**Issue by Ithanil** [bigbluebutton/bigbluebutton#17892]
**Describe the bug**
Pasting one very long (like 10000 words) or repeatedly pasting long (like 1000 words) bodies of text into the shared notes editor leads to an infinite loop of "Connected" popups and eventually to "You do not have permission to access this pad".

Using this method I have also managed to break the shared notes completely for the whole server for a certain amount of time, but I struggle to reproduce that at the moment.

**To Reproduce**
1. Generate a "lorem ipsum" text of 10000 words and store it in clipboard
2. Open shared notes, paste text
3. Observe the repeated "Connected" popups and potentially the message "You do not have permission to access this pad"

**Expected behavior**
If entering too long texts is somehow an issue, it should be catched without crashing the editor for the user or potentially breaking the shared notes feature entirely for the conference or even whole server (?).

**BBB version:**
BigBlueButton Server 2.6.6 (599)

**Comment by KlausKnopper** [comment]
It seems to matter how fast texts are inserted. Copying 8 x 1000 words sequential doesn't seem to trigger the behavior, while pasting 8000 words does.

**Comment by Ithanil** [comment]
Yes, it's about the rate of input, in terms of characters per second. If you paste 1000 words repeatedly in very quick succession, it does trigger the behavior. Doing the same with very short texts does not.

**Comment by Ithanil** [comment]
The corresponding issue in Etherpad repo: https://github.com/ether/etherpad-lite/issues/4951

Workaround is to increase "maxHttpBufferSize" in /usr/share/etherpad-lite/settings.json . Credits to @JeanPluzo for digging it out from the Etherpad issue.

**Comment by JeanPluzo** [comment]
I knew I saw the issue already.
Thanks @Ithanil !

**Comment by Sarrus** [comment]
I've had this issue as well, many thanks for the workaround!
